src/tests/stubs/D3MTestPool.sol

```diff
@@ -144,7 +149,7 @@ contract D3MTestPool is ID3MPool {
     }
 
     function maxWithdraw() external view override returns (uint256) {
-        return TokenLike(dai).balanceOf(share);
+        return _min(TokenLike(dai).balanceOf(share), assetBalance());
```

Comment:
Can you explain this change please.

Reply:
I had a test failing due this. The correct behaviour of `maxWithdraw` should be the minimum between the shares of DAI owned by our pool contract in the third party pool and the total available DAI in that moment in the whole pool. So basically now is doing what the real pools do.

Reply:
This was the test failing: `test_wind_limited_by_pool_loss` and it makes sense as share decreased from our balance.
Which btw I'm now renaming to: `test_unwind_due_to_by_pool_loss`

Reply:
perfect